### ISSUE TITLE
2025-01-09, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#46]
# Today's Arxiv Papers

(1) [Re-ranking the Context for Multimodal Retrieval Augmented Generation](http://arxiv.org/abs/2501.04695v1), 2025-01-08

(2) [EpiCoder: Encompassing Diversity and Complexity in Code Generation](http://arxiv.org/abs/2501.04694v1), 2025-01-08

(3) [URSA: Understanding and Verifying Chain-of-thought Reasoning in Multimodal Mathematics](http://arxiv.org/abs/2501.04686v1), 2025-01-08

(4) [Towards System 2 Reasoning in LLMs: Learning How to Think With Meta Chain-of-Though](http://arxiv.org/abs/2501.04682v1), 2025-01-08

(5) [Enhancing Financial VQA in Vision Language Models using Intermediate Structured Representations](http://arxiv.org/abs/2501.04675v1), 2025-01-08

(6) [Are They the Same? Exploring Visual Correspondence Shortcomings of Multimodal LLMs](http://arxiv.org/abs/2501.04670v1), 2025-01-08

(7) [Assessing Language Comprehension in Large Language Models Using Construction Grammar](http://arxiv.org/abs/2501.04661v1), 2025-01-08

(8) [Multi-task retriever fine-tuning for domain-specific and efficient RAG](http://arxiv.org/abs/2501.04652v1), 2025-01-08

(9) [FlairGPT: Repurposing LLMs for Interior Designs](http://arxiv.org/abs/2501.04648v1), 2025-01-08

(10) [Knowledge Retrieval Based on Generative AI](http://arxiv.org/abs/2501.04635v1), 2025-01-08

(11) ["Can you be my mum?": Manipulating Social Robots in the Large Language Models Era](http://arxiv.org/abs/2501.04633v1), 2025-01-08

(12) [Quantum-inspired Embeddings Projection and Similarity Metrics for Representation Learning](http://arxiv.org/abs/2501.04591v1), 2025-01-08

(13) [InfiGUIAgent: A Multimodal Generalist GUI Agent with Native Reasoning and Reflection](http://arxiv.org/abs/2501.04575v1), 2025-01-08

(14) [OpenOmni: Large Language Models Pivot Zero-shot Omnimodal Alignment across Language with Real-time Self-Aware Emotional Speech Synthesis](http://arxiv.org/abs/2501.04561v1), 2025-01-08

(15) [The Impostor is Among Us: Can Large Language Models Capture the Complexity of Human Personas?](http://arxiv.org/abs/2501.04543v1), 2025-01-08

(16) [rStar-Math: Small LLMs Can Master Math Reasoning with Self-Evolved Deep Thinking](http://arxiv.org/abs/2501.04519v1), 2025-01-08

(17) [CGP-Tuning: Structure-Aware Soft Prompt Tuning for Code Vulnerability Detection](http://arxiv.org/abs/2501.04510v1), 2025-01-08

(18) [Integrating remote sensing data assimilation, deep learning and large language model for interactive wheat breeding yield prediction](http://arxiv.org/abs/2501.04487v1), 2025-01-08

(19) [When LLMs Struggle: Reference-less Translation Evaluation for Low-resource Languages](http://arxiv.org/abs/2501.04473v1), 2025-01-08

(20) [Hidden Entity Detection from GitHub Leveraging Large Language Models](http://arxiv.org/abs/2501.04455v1), 2025-01-08

(21) [Integrating LLMs with ITS: Recent Advances, Potentials, Challenges, and Future Directions](http://arxiv.org/abs/2501.04437v1), 2025-01-08

(22) [Federated Fine-Tuning of LLMs: Framework Comparison and Research Directions](http://arxiv.org/abs/2501.04436v1), 2025-01-08

(23) [End-to-End Bangla AI for Solving Math Olympiad Problem Benchmark: Leveraging Large Language Model Using Integrated Approach](http://dx.doi.org/10.5121/ijnlc.2024.13604), 2025-01-08

(24) [SEO: Stochastic Experience Optimization for Large Language Models](http://arxiv.org/abs/2501.04393v1), 2025-01-08

(25) [Understanding Before Reasoning: Enhancing Chain-of-Thought with Iterative Summarization Pre-Prompting](http://arxiv.org/abs/2501.04341v1), 2025-01-08

(26) [Building a Mind Palace: Structuring Environment-Grounded Semantic Graphs for Effective Long Video Analysis with LLMs](http://arxiv.org/abs/2501.04336v1), 2025-01-08

(27) [Navigating the Designs of Privacy-Preserving Fine-tuning for Large Language Models](http://arxiv.org/abs/2501.04323v1), 2025-01-08

(28) [Who Does the Giant Number Pile Like Best: Analyzing Fairness in Hiring Contexts](http://arxiv.org/abs/2501.04316v1), 2025-01-08

(29) [RoRA: Efficient Fine-Tuning of LLM with Reliability Optimization for Rank Adaptation](http://arxiv.org/abs/2501.04315v1), 2025-01-08

(30) [Your Fix Is My Exploit: Enabling Comprehensive DL Library API Fuzzing with Large Language Models](http://arxiv.org/abs/2501.04312v1), 2025-01-08

(31) [LLM4SR: A Survey on Large Language Models for Scientific Research](http://arxiv.org/abs/2501.04306v1), 2025-01-08

(32) [Multimodal Graph Constrastive Learning and Prompt for ChartQA](http://arxiv.org/abs/2501.04303v1), 2025-01-08

(33) [H-MBA: Hierarchical MamBa Adaptation for Multi-Modal Video Understanding in Autonomous Driving](http://arxiv.org/abs/2501.04302v1), 2025-01-08

(34) [Mapping the Edge of Chaos: Fractal-Like Boundaries in The Trainability of Decoder-Only Transformer Models](http://arxiv.org/abs/2501.04286v1), 2025-01-08

(35) [Separate Source Channel Coding Is Still What You Need: An LLM-based Rethinking](http://arxiv.org/abs/2501.04285v1), 2025-01-08

(36) [OpenIN: Open-Vocabulary Instance-Oriented Navigation in Dynamic Domestic Environments](http://arxiv.org/abs/2501.04279v1), 2025-01-08

(37) [Exploring the Expertise of Large Language Models in Materials Science and Metallurgical Engineering](http://arxiv.org/abs/2501.04277v1), 2025-01-08

(38) [Robotic Programmer: Video Instructed Policy Code Generation for Robotic Manipulation](http://arxiv.org/abs/2501.04268v1), 2025-01-08

(39) [Scaling Large Language Model Training on Frontier with Low-Bandwidth Partitioning](http://arxiv.org/abs/2501.04266v1), 2025-01-08

(40) [IOLBENCH: Benchmarking LLMs on Linguistic Reasoning](http://arxiv.org/abs/2501.04249v1), 2025-01-08

(41) [TransientVerse: A Comprehensive Real-Time Alert and Multi-Wavelength Analysis System for Transient Astronomical Events](http://arxiv.org/abs/2501.04247v1), 2025-01-08

(42) [Statistical Uncertainty Quantification for Aggregate Performance Metrics in Machine Learning Benchmarks](http://arxiv.org/abs/2501.04234v1), 2025-01-08

(43) [Agent Laboratory: Using LLM Agents as Research Assistants](http://arxiv.org/abs/2501.04227v1), 2025-01-08

(44) [Reasoning-Enhanced Self-Training for Long-Form Personalized Text Generation](http://arxiv.org/abs/2501.04167v1), 2025-01-07

(45) [AdaptiveCoPilot: Design and Testing of a NeuroAdaptive LLM Cockpit Guidance System in both Novice and Expert Pilots](http://arxiv.org/abs/2501.04156v1), 2025-01-07

(46) ["Yeah Right!" -- Do LLMs Exhibit Multimodal Feature Transfer?](http://arxiv.org/abs/2501.04138v1), 2025-01-07

(47) [Sa2VA: Marrying SAM2 with LLaVA for Dense Grounded Understanding of Images and Videos](http://arxiv.org/abs/2501.04001v1), 2025-01-07

(48) [RAG-Check: Evaluating Multimodal Retrieval Augmented Generation Performance](http://arxiv.org/abs/2501.03995v1), 2025-01-07

(49) [Influences on LLM Calibration: A Study of Response Agreement, Loss Functions, and Prompt Styles](http://arxiv.org/abs/2501.03991v1), 2025-01-07

(50) [(De)-Indexing and the Right to be Forgotten](http://arxiv.org/abs/2501.03989v1), 2025-01-07

(51) [VLM-driven Behavior Tree for Context-aware Task Planning](http://arxiv.org/abs/2501.03968v1), 2025-01-07

(52) [Vision Language Models as Values Detectors](http://arxiv.org/abs/2501.03957v1), 2025-01-07

(53) [Localizing AI: Evaluating Open-Weight Language Models for Languages of Baltic States](http://arxiv.org/abs/2501.03952v1), 2025-01-07

(54) [Not all tokens are created equal: Perplexity Attention Weighted Networks for AI generated text detection](http://arxiv.org/abs/2501.03940v1), 2025-01-07

(55) [Exploring the Potential of Large Language Models in Public Transportation: San Antonio Case Study](http://arxiv.org/abs/2501.03904v1), 2025-01-07

(56) [LLaVA-Mini: Efficient Image and Video Large Multimodal Models with One Vision Token](http://arxiv.org/abs/2501.03895v1), 2025-01-07

(57) [LEAP: LLM-powered End-to-end Automatic Library for Processing Social Science Queries on Unstructured Data](http://dx.doi.org/10.14778/3705829.3705843), 2025-01-07

(58) [AlphaPO -- Reward shape matters for LLM alignment](http://arxiv.org/abs/2501.03884v1), 2025-01-07

(59) [CL3DOR: Contrastive Learning for 3D Large Multimodal Models via Odds Ratio on High-Resolution Point Clouds](http://arxiv.org/abs/2501.03879v1), 2025-01-07

(60) [Progressive Document-level Text Simplification via Large Language Models](http://arxiv.org/abs/2501.03857v1), 2025-01-07

(61) [More is not always better? Enhancing Many-Shot In-Context Learning with Differentiated and Reweighting Objectives](http://arxiv.org/abs/2501.04070v1), 2025-01-07

(62) [MedFocusCLIP : Improving few shot classification in medical datasets using pixel wise attention](http://arxiv.org/abs/2501.03839v1), 2025-01-07

(63) [KAnoCLIP: Zero-Shot Anomaly Detection through Knowledge-Driven Prompt Learning and Enhanced Cross-Modal Integration](http://arxiv.org/abs/2501.03786v1), 2025-01-07

(64) [Context-Alignment: Activating and Enhancing LLM Capabilities in Time Series](http://arxiv.org/abs/2501.03747v1), 2025-01-07

(65) [ChronoLLM: A Framework for Customizing Large Language Model for Digital Twins generalization based on PyChrono](http://arxiv.org/abs/2501.04062v1), 2025-01-07

(66) [SLAM: Towards Efficient Multilingual Reasoning via Selective Language Alignment](http://arxiv.org/abs/2501.03681v1), 2025-01-07

(67) [SMIR: Efficient Synthetic Data Pipeline To Improve Multi-Image Reasoning](http://arxiv.org/abs/2501.03675v1), 2025-01-07

(68) [CommitShield: Tracking Vulnerability Introduction and Fix in Version Control Systems](http://arxiv.org/abs/2501.03626v1), 2025-01-07

(69) [LlaMADRS: Prompting Large Language Models for Interview-Based Depression Assessment](http://arxiv.org/abs/2501.03624v1), 2025-01-07

(70) [From Code to Compliance: Assessing ChatGPT's Utility in Designing an Accessible Webpage -- A Case Study](http://arxiv.org/abs/2501.03572v1), 2025-01-07

(71) [What Does a Software Engineer Look Like? Exploring Societal Stereotypes in LLMs](http://arxiv.org/abs/2501.03569v1), 2025-01-07

(72) [Applying Large Language Models in Knowledge Graph-based Enterprise Modeling: Challenges and Opportunities](http://arxiv.org/abs/2501.03566v1), 2025-01-07

(73) [Bridged Semantic Alignment for Zero-shot 3D Medical Image Diagnosis](http://arxiv.org/abs/2501.03565v1), 2025-01-07

(74) [Beyond Factual Accuracy: Evaluating Coverage of Diverse Factual Information in Long-form Text Generation](http://arxiv.org/abs/2501.03545v1), 2025-01-07

(75) [PromptGuard: Soft Prompt-Guided Unsafe Content Moderation for Text-to-Image Models](http://arxiv.org/abs/2501.03544v1), 2025-01-07

(76) [Deep Learning for Pathological Speech: A Survey](http://arxiv.org/abs/2501.03536v1), 2025-01-07

(77) [SenseRAG: Constructing Environmental Knowledge Bases with Proactive Querying for LLM-Based Autonomous Driving](http://arxiv.org/abs/2501.03535v2), 2025-01-07

(78) [A Sequential Optimal Learning Approach to Automated Prompt Engineering in Large Language Models](http://arxiv.org/abs/2501.03508v1), 2025-01-07

(79) [Can LLMs Design Good Questions Based on Context?](http://arxiv.org/abs/2501.03491v1), 2025-01-07

(80) [Entropy-Guided Attention for Private LLMs](http://arxiv.org/abs/2501.03489v1), 2025-01-07

(81) [Align-Pro: A Principled Approach to Prompt Optimization for LLM Alignment](http://arxiv.org/abs/2501.03486v1), 2025-01-07

(82) [Women, Infamous, and Exotic Beings: What Honorific Usages in Wikipedia Reveal about the Socio-Cultural Norms](http://arxiv.org/abs/2501.03479v1), 2025-01-07

(83) [Reading with Intent -- Neutralizing Intent](http://arxiv.org/abs/2501.03475v1), 2025-01-07

(84) [MTRAG: A Multi-Turn Conversational Benchmark for Evaluating Retrieval-Augmented Generation Systems](http://arxiv.org/abs/2501.03468v1), 2025-01-07

(85) [ISSR: Iterative Selection with Self-Review for Vocabulary Test Distractor Generation](http://arxiv.org/abs/2501.03462v1), 2025-01-07

(86) [Activating Associative Disease-Aware Vision Token Memory for LLM-Based X-ray Report Generation](http://arxiv.org/abs/2501.03458v1), 2025-01-07

(87) [CoReQA: Uncovering Potentials of Language Models in Code Repository Question Answering](http://arxiv.org/abs/2501.03447v1), 2025-01-07

(88) [LLM4CVE: Enabling Iterative Automated Vulnerability Repair with Large Language Models](http://arxiv.org/abs/2501.03446v1), 2025-01-07

(89) [Finding A Voice: Evaluating African American Dialect Generation for Chatbot Technology](http://arxiv.org/abs/2501.03441v1), 2025-01-07
